### DISCUSSION TITLE
GET API call max records in response

### DISCUSSION THREAD

**Discussion by ShivaniSK** [yahoo/elide#3108] [Q&A] [answered]
Hi,
What are the max number of records which a GET API returns in response?
Currently my data source contains around 2k records, however i can see that not all are returned in the GET response.
Is there any pagination going on? How to fetch the remaining records?

Regards,
Shivani

**Comment by justin-tay** [comment, marked as answer]
There is a default page size of 500 if none is provided in the request.

You can either configure the default page size higher or set a page size for the request.

For JSON-API this can be done by setting the `page[limit]` query parameter.

`http://localhost:8080/api/group?page%5Blimit%5D=3000`

You can refer to the documentation for [JSON API](https://elide.io/pages/guide/v7/10-jsonapi.html) for the syntax for pagination.
